### PR TITLE
modify servlet to store comments

### PR DESCRIPTION
**Pull request by erysjd**
I modified the servlet file to strore the comments in datastore.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -53,6 +53,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       comments.add(comment);
       response.sendRedirect("/index.html");
```
Comment (mkruskal-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: You should return early here, otherwise you'll still store an empty comment

Comment (erysjd) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: What do you mean exactly? Return what early?

Comment (mkruskal-google) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: just return void (i.e. "return;").

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -47,11 +47,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Get the input from the form.
     String comment = getParameter(request, "text-input", "");
-    if (comment != ""){
-      comments.add(comment);
-      response.sendRedirect("/index.html");  
+    if (comment == ""){
+        return;
+    }
+    else{
```
Comment (mkruskal-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: This else isn't necessary, since the return will exit the function

**Pull request merged**